Dockerfile

```diff
@@ -89,14 +89,17 @@ RUN apt update && \
 # Install SQLite 3.x
 # https://www.sqlite.org/download.html
 # https://www.sqlite.org/howtocompile.html#compiling_the_command_line_interface
+ADD .patch /tmp
```

Comment:
I think `COPY` tends to be best, per https://stackoverflow.com/a/24958548. (And what we use elsewhere.)